packages/core/src/InstanceSettings.ts

```diff
@@ -61,6 +71,22 @@ export class InstanceSettings {
 	 */
 	instanceRole: InstanceRole = 'unset';
 
+	/**
+	 * Transient ID of this n8n instance, for scaling mode.
+	 * Reset on restart. Do not confuse with `instanceId`.
+	 *
+	 * @example 'main-bnxa1riryKUNHtln'
+	 * @example 'worker-nDJR0FnSd2Vf6DB5'
+	 * @example 'webhook-jxQ7AO8IzxEtfW1F'
+	 */
+	hostId = '';
```

Comment:
since `instanceType` is now already being set in the constructor, we could not change this to 
```suggestion
	readonly hostId = `${this.instanceType}-${nanoid()}`;
```
and remove `setHostId`

Reply:
[4e5cc32](https://github.com/n8n-io/n8n/pull/11262/commits/4e5cc32479dd1bf61e50d42fd9dc59ed5dbd653a)